examples/xnnpack/aot_compiler.py

```diff
@@ -103,18 +101,16 @@
             _check_ir_validity=False if args.quantize else True,
             _skip_dim_order=True,  # TODO(T182187531): enable dim order in xnnpack
         ),
+        generate_etrecord=args.etrecord is not None,
```

Comment:
Add default as `False` above 
```suggestion
        generate_etrecord=args.etrecord is not False,
```